src/main/java/com/flowable/flowableboot/service/UmsTaskService.java

```diff
@@ -0,0 +1,68 @@
+package com.flowable.flowableboot.service;
+
+import com.flowable.flowableboot.model.UmsTask;
+import com.flowable.flowableboot.repository.UmsTaskRepository;
+import org.flowable.engine.RuntimeService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+
+@Service
+public class UmsTaskService {
+    private UmsTaskRepository umsTaskRepository;
+    private RuntimeService runtimeService;
+
+    @Autowired
+    public UmsTaskService(RuntimeService runtimeService, UmsTaskRepository umsTaskRepository){
+        this.runtimeService = runtimeService;
+        this.umsTaskRepository = umsTaskRepository;
+    }
+
+    /**
+     * Create a new UmsTask - Ideally, this would assume a process was started in workflow engine
+     *
+     * @param process_instance_id process_instance_id
+     * @param name name
+     * @param requester requester
+     * @param assignee assignee
+     * @param priority priority
+     * @param dueDate dueDate
+     * @param receivedDate receivedDate
+     * @param loe loe
+     * @param status status
+     * @param description description
+     *
+     * @return UmsTask Entity
+     */
+    public UmsTask createUmsTask(String process_instance_id, String name, String requester, String assignee, Integer priority,
+                                 LocalDateTime dueDate, LocalDateTime receivedDate, String loe, String  status, String description){
+
+        // don't know if we want to abstract this more - may need a task and process service
```

Comment:
We will likely want separate services as this grows, but it is not necessary for this task.

Reply:
okay...I was wondering if this check should be at the controller level instead of the service